soepy/test/test_regression.py

```diff
@@ -21,6 +21,10 @@ def test1():
 
         model_spec_init_dict, random_model_params_df, expected_df = tests[i]
 
+        random_model_params_df.loc[
```

Comment:
Please add a TODO here that this is just a fix to maintain the current regression vault. Right?